api-server/Cargo.toml

```diff
@@ -21,3 +21,4 @@ serde = "1.0"
 serde_json = "1.0"
 r2d2 = "0.8"
 uuid = { version = "0.6", features = ["serde", "v4"] }
+rand = "0.8.3"
```

Comment:
So we have context behind each dependency we add can you insert a comment above this line along the lines:

```
# Generating station keys within the StationsModel
```